src/main/webapp/data/userMapData.json

```diff
@@ -1489,10 +1529,12 @@
       "Truman State University",
       "Tufts University",
       "UC San Diego",
+      "UNC - Chapel Hill",
```

Comment:
This should be moved to after the next line to maintain alphabetical order. Also, the dash can be dropped for consistency with `UNC Pembroke`.